ingest-asset-reconciler/src/test/scala/uk/gov/nationalarchives/testUtils/ExternalServicesTestUtils.scala

```diff
@@ -84,6 +85,12 @@ class ExternalServicesTestUtils(dynamoServer: WireMockServer) extends TableDrive
        |      "digitalAssetSubtype": {
        |        "S": "Test Digital Asset Subtype"
        |      },
+       |      "representationType": {
+       |        "S": "Access"
+       |      },
+       |      "representationSuffix": {
+       |        "N": "2"
```

Comment:
Even though it doesn't really matter since it's a test, shouldn't it be "1" since we start at 1 for each RepresentationType?